typescript/src/lib/moderation/structures/ModerationMessageEvent.ts

```diff
@@ -160,11 +166,12 @@ export abstract class ModerationMessageEvent<T = unknown> extends Event {
 		unlock();
 	}
 
-	protected onLog(message: GuildMessage, language: TFunction, value: T): Awaited<void> {
+	protected onLog(message: GuildMessage, channelId: string | Nullish, language: TFunction, value: T): Awaited<void> {
```

Comment:
```suggestion
	protected onLog(message: GuildMessage, logChannelId: string | Nullish, language: TFunction, value: T): Awaited<void> {
```

And update references